tests/bud.bats

```diff
@@ -1981,6 +1981,26 @@ _EOF
   test -s $root/etc/passwd
 }
 
+@test "build using --layer-label and test labels on intermediate images" {
+  # Remove all images so no intermediate images are present
+  run_buildah rmi --all -f
+  _prefetch alpine
+
+  run_buildah build --no-cache --layers --layer-label somefancylayerlabel=true $WITH_POLICY_JSON -t exp -f $BUDFILES/simple-multi-step/Containerfile
+
+  # Final image must not contain this label
+  run_buildah inspect --format '{{ index .Docker.Config.Labels "somefancylayerlabel"}}' exp
+  expect_output "" "label label on actual image"
+
+  # All intermediate images must have this label
+  run_buildah images -a --format '{{.ID}}' --filter intermediate=true
+  for image in "${lines[@]}";do
+  run_buildah inspect --format '{{ index .Docker.Config.Labels "somefancylayerlabel"}}' $image
+	expect_output "true" "somefancylayerlabel on intermediate image"
```

Comment:
Please include `$image` in the failure message, e.g. `"somefancylayerlabel on intermediate image '$image'"`, that makes it much easier for a test investigator to see what happened on failure.

Reply:
fixed but now match is changed.